.github/workflows/mcad-CI.yml

```diff
@@ -0,0 +1,37 @@
+# This workflow will build mcad and run unit test and E2E tests
+name: MCAD-CI
+on:
+  pull_request_target:
+    types: [ opened, labeled, unlabeled, synchronize ]
```

Comment:
Do you need to specify specific event types?
I think that default types should be sufficient. If so, can you remove these explicit types?

Reply:
Yes Default types is enough now